example/src/main/java/io/el/example/executor/TaskExecutor.java

```diff
@@ -27,25 +28,24 @@ protected void run() {
     }
   }
 
-  public static void main(String[] args) throws InterruptedException {
-    TaskExecutor scheduler = new TaskExecutor(
-        new ThreadPerTaskExecutor(Executors.defaultThreadFactory()));
-    int MAX_DELAY = 10000;
-
-    long start = System.currentTimeMillis();
-    for (int i = 0; i < 1000; i += 1) {
-      long innerStart = System.currentTimeMillis();
-      try {
-        scheduler.execute(new SimpleTask(String.valueOf(i), 500));
-      } catch (RejectedExecutionException e) {
-        LOGGER.info("Thread[{}] - Executed failed. Task ID [{}] - {} ms", Thread.currentThread().getName(), i, System.currentTimeMillis() - innerStart);
-      }
+  private static class Example extends AbstractSingleThreadEventLoopExample {
+
+    public Example(SingleThreadEventLoop eventLoop) {
+      super(eventLoop);
     }
 
-    LOGGER.info("Thread[{}] - Executed all tasks - {} ms", Thread.currentThread().getName(), System.currentTimeMillis() - start);
-    Thread.sleep(2 * MAX_DELAY);
+    @Override
+    protected void testEventLoop(String taskId, long scheduledDelayMillis) {
```

Comment:
I have a question about `scheduledDelayMillis` parameter.
`testEventLoop` method doesn't use `scheduledDelayMillis`. 

What kind of history has this abstract method?(`abstract protected void testEventLoop(String taskId, long scheduledDelayMillis);`)